packages/client/src/DataCommonsClient.ts

```diff
@@ -0,0 +1,480 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
nit: This file is quite large, would be nice if you could split out some functions. Maybe the pure data fetches can go in their own file? May can split out some of the helpers?

Reply:
Split into two classes - `DataCommonsWebClient` now only makes API calls, and `DataCommonsClient` has the public getCsv/getDataRows/getGeoJSON. I kept the helpers in `DataCommonsClient` for now since the file seems to be a manageable size.